app/tests/pages/landing.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Landing from '../../pages/landing';
+import { mount } from 'enzyme';
+import { render, fireEvent, cleanup } from '@testing-library/react';
```

Comment:
It appears that this is not in our dependencies (anymore?)./ Take a look at the other pages to see how to render this